### PR TITLE
fix: further comment nits

### PR DESCRIPTION
**Pull request by chrismaree**
PR to further refine some comments and lining nits.

### CODE REVIEW

File: programs/svm-spoke/src/instructions/fill.rs [hunk 1]
```diff
@@ -116,7 +116,7 @@ pub fn fill_v3_relay<'info>(
     fill_status_account.status = FillStatus::Filled;
     fill_status_account.relayer = *ctx.accounts.signer.key;
 
-    if relay_data.message.len() > 0 {
+    if !relay_data.message.is_empty() {
```
Comment (md0x) [inline programs/svm-spoke/src/instructions/fill.rs:119]: 👌

File: programs/svm-spoke/src/event.rs [hunk 2]
```diff
@@ -30,6 +30,11 @@ pub struct RelayedRootBundle {
     pub slow_relay_root: [u8; 32],
 }
 
+#[event]
+pub struct EmergencyDeleteRootBundle {
```
Comment (Reinis-FRP) [inline programs/svm-spoke/src/event.rs:34]: didn't we rename this?

Comment (chrismaree) [reply programs/svm-spoke/src/event.rs:34]: done.

**Review by Reinis-FRP — APPROVED** [review]
nice!

**Pull request merged**